docs/source/array-overview.rst

```diff
@@ -79,6 +78,25 @@ supports numpy-style item assignment like an ``h5py.Dataset``.
    >>> y.store(dset)
 
 
+On-Disk Storage
+---------------
+
+In the example above we used ``h5py`` but ``dask.array`` works equally well
+with ``pytables``, ``bcolz``, or any library that provides an array object from
+which we can slice out numpy arrays.
+
+.. code-block:: Python
+
+   >>> x = ooc[1000:2000, :2000]  # pull out numpy array from on-disk object
+
+This API has become a standard in Scientific Python.  Dask works with any
+object that supports this operation and the equivalent assignment syntax.
+
+.. code-block:: Python
+
+   >>> ooc[1000:2000, :2000] = x  # Store numpy array in on-disk object
```

Comment:
What does `ooc` stand for? I think it would be more interesting to have an explicit example.